.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "files.exclude": {
+    ".mypy_cache": true,
+    "**/*.egg-info": true,
+    "**/Network_*.log": true,
+    "**/Network_*.txt": true,
+    "**/*.pyc": { "when": "$(basename).py" },
+    "**/__pycache__": true,
+    "**/.venv": true
```

Comment:
I also use `env`, so might as well include the same that the `.gitignore` does:

```
.env
.venv
env/
venv/
ENV/
env.bak/
venv.bak/
```

Reply:
Hm, `env` could be spicy due to actual env files, I thought you used `venv` tho? We should probably make this consistent with `nari` itself. Prefer to keep backup folders around unless their base exists, so it's clear when we have leftovers.